### PR TITLE
OG-1075: ContractInteractor causes 'operation=getAddress code=UNSUPPORTED_OPERATION'

### PR DESCRIPTION
**Pull request by forshtat**

### CODE REVIEW

**Comment by Nicomendia** [comment]
Hello, some ETA for this PR merging and new version release? Thanks in advance!

**Review by galimba — COMMENTED** [review]
@forshtat Can we have a short description of the changes this PR introduces, and how the issue is fixed?

File: packages/common/src/ContractInteractor.ts [hunk 1]
```diff
@@ -178,6 +179,7 @@ export class ContractInteractor {
     this.versionManager = versionManager ?? new VersionsManager(gsnRuntimeVersion, gsnRequiredVersion)
     this.deployment = deployment
     this.provider = provider
+    this.signer = signer
```
Comment (drortirosh) [inline packages/common/src/ContractInteractor.ts:182]: AFAIK, by having `this.signer = signer ?? provider.getSigner()`
you make it an optional parameter, and thus remove most of the needed changes below.

**Comment by viac92** [comment]
Any update for the new release?

**Comment by Ljrr3045** [comment]
Hello, I wanted to know if there is any update about this new release? any release date?

**Comment by forshtat** [comment]
> Hello, I wanted to know if there is any update about this new release? any release date?

Apologies, it slipped my attention that this is still hanging, will publish a fix this weekend.

**Pull request merged**